app/views/admin/order_cycles/checkout_options.html.haml

```diff
@@ -3,7 +3,7 @@
 - content_for :page_title do
   = t :edit_order_cycle
 
-= form_for [main_app, :admin, @order_cycle], html: { class: "order_cycle" } do |f|
+= form_for [main_app, :admin, @order_cycle], html: { class: "order_cycle" , data: { controller: 'unsaved-changes', action: 'beforeunload@window->unsaved-changes#leavingPage', 'unsaved-changes-changed': "false" } } do |f|
```

Comment:
> `, 'unsaved-changes-changed': "false"`

Is it necessary to specify this in the view, or can we just let the controller initialisation set up the default value?

Reply:
We could let the controller initialise the default value, but having the ability to set it in the view gives you the option to force the warning. 
Now that I think about it :thinking: , maybe it would create a confusing user experience. You could have the situation where a user land on form, doesn't change anything and then get the warning when trying to leave the page. Happy to change that if you think that's an issue.

Reply:
I was actually wondering if this snippet could simply be removed in this case, and let the default apply. But I don't think it's necessary to change.